src/main/kotlin/nomic/api/RulesAmendments.kt

```diff
@@ -0,0 +1,42 @@
+package nomic.api
+
+import nomic.data.EntityNotFoundException
+import nomic.domain.entities.RulesAmendmentsModel
+import nomic.domain.rulesamendments.RuleAmendmentDomain
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("api")
```

Comment:
If this controller is for rules and amendments, couldn't the controller mapping for api/rules_amendments? Otherwise, this could potentially cause issues with other controllers